app/main/index.js

```diff
@@ -103,7 +103,7 @@ function checkConnectivity() {
 function checkConnection() {
 	// eslint-disable-next-line no-unused-vars
 	mainWindow.webContents.on('did-fail-load', (event, errorCode, errorDescription, validatedURL) => {
-		if (errorDescription === 'ERR_INTERNET_DISCONNECTED' || errorDescription === 'ERR_PROXY_CONNECTION_FAILED') {
+		if (errorDescription === 'ERR_INTERNET_DISCONNECTED' || errorDescription === 'ERR_PROXY_CONNECTION_FAILED' || errorDescription === 'ERR_CONNECTION_RESET' || errorDescription === 'ERR_NAME_NOT_RESOLVED') {
```

Comment:
```javascript
const connectivityERR = [  'ERR_INTERNET_DISCONNECTED',
			'ERR_PROXY_CONNECTION_FAILED',
			'ERR_CONNECTION_RESET',
			'ERR_NOT_CONNECTED',
			'ERR_NAME_NOT_RESOLVED'
		];

function checkConnection() {
	// eslint-disable-next-line no-unused-vars
	mainWindow.webContents.on('did-fail-load', (event, errorCode, errorDescription, validatedURL) => {
		const hasConnectivityErr = (connectivityERR.indexOf(errorDescription) >= 0);
		if (hasConnectivityErr) {
			console.error('error', errorDescription)
			checkConnectivity();
		}
	});
}
```
Added `hasConnectivityErr` const.
We can use this constant later too in other issues if needed.